tower_cli/resources/inventory_source.py

```diff
@@ -47,6 +47,7 @@ class Resource(models.MonitorableResource):
     # Only used if update_on_launch is used
     update_cache_timeout = models.Field(type=int, required=False,
                                         display=False)
+    timeout =  models.Field(type=int, required=False, display=False)
```

Comment:
How about some help text telling what the integer should be - seconds vs. minutes